AGA8CODE/JS/Globals.js

```diff
@@ -0,0 +1,86 @@
+let RGERG;

```

Comment:
This file is included two places, should one be removed?

Reply:
I just copy the way was presented in VBA. An implementation in one side and the brute code in the other.
I'm afraid that if i remove the code from the example the end user will need to modify the index.html and index.js in a way to make it work.
Of course, another course of action will be to remove the example and adding more detailed documentation. If you wish i began to work on it. (even if the example remains, adding more documentation will be a good idea)
Regarding the textArea. Is just a space were i can read the gas composition in a less tedious way for the user, directly from a spreadsheet. As soon as the JS code reads the data, the textArea is erased again.
![ChartArea](https://user-images.githubusercontent.com/71098233/119730811-8bf7d900-be4c-11eb-8a13-29f953cb304a.png)

Reply:
I see what you mean here, but I had no idea what you intended when I opened index.html. If you wanted to add a few lines, that would be good.

Reply:
Ian:
After your comments, I've started a new Index more modern, and user friendly.
I'll ask you for a few days to accomplish the new Example Page.
Hope this is Ok.
Best Regards.

Reply:
No problem at all. No rush on my side.